test/OpenTelemetry.Instrumentation.AWSLambda.Tests/Implementation/AWSLambdaHttpUtilsTests.cs

```diff
@@ -195,7 +195,11 @@ public void SetHttpTagsFromResult_APIGatewayProxyResponse_SetsCorrectTags()
         {
             { "http.status_code", 200 },
         };
-        AssertTags(expectedTags, activity.TagObjects);
+
+        var actualTags = activity?.TagObjects
```

Comment:
Shouldn't a null activity cause an error immediately? Otherwise it will look like an activity was created but with empty tags. I also don't think converting null values to empty objects is good. This should not need any conversion and AssertTags should cause an assertion failure if it affects one of the checked attributes.

Reply:
Added null assertion for the `activity`.